### PR TITLE
Add Download option to Resource

### PR DESCRIPTION
**Pull request by dhaneshnm**
Enable download option in Resource. This will allow down the resource
meta data in all the formats supported by the API.

Fixes BlueBrain/nexus/issues/900

### CODE REVIEW

File: src/shared/views/ResourceView.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ import * as queryString from 'query-string';
 import { useAsyncEffect } from 'use-async-effect';
 import { useNexusContext } from '@bbp/react-nexus';
 import { Resource, ResourceLink } from '@bbp/nexus-sdk';
-
 import { getResourceLabel } from '../utils';
```
Comment (dhaneshnm) [inline src/shared/views/ResourceView.tsx]: These are added by prettier.

File: src/shared/containers/ResourceDownloadContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+type formatType = 'vnd.graph-viz' | 'n-triples' | 'json';
+
+const ResourceDownloadButton: React.FunctionComponent<{
+  orgLabel: string;
+  projectLabel: string;
+  resourceId: string;
+}> = props => {
+  const nexus = useNexusContext();
+  console.log(props.orgLabel);
```
Comment (Stafeeva) [inline src/shared/containers/ResourceDownloadContainer.tsx]: Do we need to log it?

File: src/shared/containers/ResourceDownloadContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        const blob = new Blob([
+          format === 'json' ? JSON.stringify(result) : result.toString(),
+        ]);
+        const url = window.URL.createObjectURL(blob);
+        setDownloadUrl(url);
+        setFilename(`${resourceId}.${extension}`);
+        refContainer.current.click();
+      })
+      .catch(error => {
+        console.log(error);
```
Comment (Stafeeva) [inline src/shared/containers/ResourceDownloadContainer.tsx]: Forgotten console.log?

**Comment by dhaneshnm** [comment]
Screenshot_2020-01-15 A knowledge graph for data-driven science

**Pull request merged**